go.mod

```diff
@@ -3,30 +3,38 @@ module github.com/google/generative-ai-go
 go 1.21
 
 require (
```

Comment:
How did you get this change? From `go mod tidy`? I wonder where the go-logr dependencies came from? They weren't there before.

I think we should remove these changes from this PR and do another one that is just the result of `go mod tidy`.